fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -676,6 +676,21 @@ private void analyzeAndGenerateQueryPlan(TQueryOptions tQueryOptions) throws Use
                 }
                 if (explainOptions != null) parsedStmt.setIsExplain(explainOptions);
             }
+
```

Comment:
Why do you need to set explain option repeatedly here?

Reply:
deleted